src/mlocale.h

```diff
@@ -28,7 +28,9 @@
 #include <QMap>
 
 class QString;
-class QStringList;
+#if QT_VERSION < 0x051500
```

Comment:
Why this? All versions have QStringList in the API and even if those didn't, the class declaration shouldn't have too much side-effects?